recipes-core/updatehub/updatehub-config-toradex/unlock_emmc

```diff
@@ -1,9 +1,9 @@
 #!/bin/sh
 
 if [ "$1" = "install" ]; then
-    echo 0 > /sys/block/mmcblk0boot0/force_ro
+    echo 0 > /sys/block/mmcblk1boot0/force_ro
```

Comment:
Commit log:

updatehub-config-toradex: Fix block device name in unlock_emmc script